### ISSUE TITLE
[IC2 Classic] Resourcepack not used by BlueMap

### ISSUE THREAD

**Issue by victor-siu** [BlueMap-Minecraft/BlueMap#475]
IC2 Classic uses a different structure of resourcepack as [mentioned in their wiki](https://github.com/TinyModularThings/IC2Classic/wiki/1.19.x-ResourcePacks).

I added the resourcepack ([IC2U ResourcePack.zip](https://github.com/TinyModularThings/IC2Classic/files/9872435/IC2U.ResourcePack.zip)), provided by the mod dev team, to the directory `config/bluemap/resourcepacks`. However, it is not used by BlueMap after `/bluemap purge overworld`, causing missing texture on the online map.

![The missing texture is rubber tree.](https://github.com/BlueMap-Minecraft/BlueMap/assets/7189931/4f28162e-c241-449c-8001-954e0e30e73f)
The missing texture is rubber tree.

**Comment by TBlueF** [comment]
The first line on the page you linked says:
> IC2Classic uses custom JsonModels that load via code, [...]

Which means they don't provide models at all, but generate them at runtime. They only provide textures.
BlueMap needs those json-models to figure out how a block looks and which textures to use. As BlueMap can't just run another mod's code, there is no other way to figure out the block-models.

If you want IC2 Blocks to render, you will need to provide the block-models in form of a normal ResourcePack (in bluemaps [resourcepacks folder](https://bluemap.bluecolored.de/wiki/customization/ResourcePacks.html)).

**Tl;dr:** Adding support for their resourcepack-format won't solve the problem, since it doesn't contain block-models. Closing this issue :)
